subgraph/schema.graphql

```diff
@@ -117,3 +119,14 @@ type Receipt @entity {
   "Increasing number of receipt"
   receiptNumber: BigInt!
 }
+
+type ReferrerFee @entity {
+  "Transaction Hash"
+  id: ID!
+  "Address of the referrer."
+  referrer: Bytes!
+  "Fee attributed to the referrer. E.g., 200 represents 20%."
+  fee: BigInt!
+  "In the Unlock ecosystem, a “Lock” is a smart contract that creates (or “mints”) NFTs"
+  lock: Lock!
+}
```

Comment:
Here is the `ReferrerFee` entity. Similar to what we do with `LockStats` entity or `Receipt` entity:

id - `Transaction Hash`
referrer - `The address you gave the referral to`
fee - `Fee you gave to the referrer`
lock - `The lock the which referrer fee belongs to`

![image](https://github.com/unlock-protocol/unlock/assets/136458308/3b7e907d-f4dc-422a-9e8e-15aba5a73ce1)

Reply:
I am a bit confused about the use of the transaction hash? Is that the transaction that "sets" the fee? What happens if the lock manager changes the fee then? do you remove the `ReferrerFee` or change it?

I think a better model would be to just use the `referrer` address as the `id` since a referrer fee is unique per lock and referrer address.

Reply:
Good idea. I wasn't aware of that lock manager can change the fee for a given address. Let me change that 👍

Reply:
Fixed 👍